cmd/marathon-resource/behaviors/template.go

```diff
@@ -30,3 +36,23 @@ func parsePayload(p Params, path string) (io.Reader, error) {
 	}
 	return buf, nil
 }
+
+func replaceStrings(metadata []Metadata, replacements map[string]string) map[string]string {
+	for _, v := range metadata {
+		replacements[v.Name] = v.Value
+	}
+
+	return replacements
+}
+
+func replaceFiles(metadata []Metadata, replacements map[string]string, path string) (map[string]string, error) {
+	for _, v := range metadata {
+		fileValue, err := ioutil.ReadFile(filepath.Join(path, v.Value))
+		if err != nil {
+			return replacements, fmt.Errorf("Error replaceing %s from replacement_files: %v", v.Name, err)
+		}
+		replacements[v.Name] = strings.TrimSpace(string(fileValue))
+	}
+
+	return replacements
```

Comment:
It's not compiling, this line needs to be `return replacements, nil`. Mind fixing it?